### PR TITLE
MUMUP-2062: Notification style

### PR DESCRIPTION
**Pull request by jhanstra**
Includes:
- More stylish notifications
- x instead of garbage icon
- Tooltip on hover over dismiss icon

In Action:
screen recording 2015-09-02 at 06 12 pm

### CODE REVIEW

File: src/main/webapp/css/buckyless/notifications.less [hunk 1]
```diff
@@ -1,140 +1,179 @@
...
+    font-size: 1.1em;
+    color: #333;
+    background-color: #e7d9c1;
+    border-radius: 4px;
+    margin: 8px 0px;
+    padding: 8px 10px 8px 10px;
+    border:none;
+    border-left: 10px solid @color1;
+    font-weight: 400;
+  } */
```
Comment (timlevett) [inline src/main/webapp/css/buckyless/notifications.less]: I think we should just delete this. We can always get it from history if we really really really want it :)

File: src/main/webapp/portal/notifications/partials/notifications-full.html [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
-            <a ng-href='{{notification.actionURL}}' target='_blank' aria-label='{{notification.actionAlt}}'>{{ notification.title }}</a>
-            <button class='btn btn-flat' ng-click='dismiss(notification)' ng-hide="!notification.dismissable || isDismissed(notification)">
-              <i class='fa fa-trash'></i>
-            </button>
+						<a class="content" ng-href='{{notification.actionURL}}' target='_blank' aria-label='{{notification.actionAlt}}'>
+	            <p>{{ notification.title }}</p>
+						</a>
+						<div class="action" ng-click='dismiss(notification)' ng-hide="!notification.dismissable || isDismissed(notification)">
+	            <i class='fa fa-times'tooltip="Dismiss notification" tooltip-placement="top" tooltip-popup-delay="300" tooltip-trigger="mouseenter"></i>
+						</div>
```
Comment (timlevett) [inline src/main/webapp/portal/notifications/partials/notifications-full.html]: tab vs space?

**Comment by timlevett** [comment]
:+1: Looks great @jhanstra

**Comment by keirserrie** [comment]
looks great :+1:

File: src/main/webapp/css/buckyless/notifications.less (hunk 1 above)
Comment (jhanstra) [inline src/main/webapp/css/buckyless/notifications.less]: Agreed, I'll delete

File: src/main/webapp/portal/notifications/partials/notifications-full.html (hunk 2 above)
Comment (jhanstra) [inline src/main/webapp/portal/notifications/partials/notifications-full.html]: Ugh why don't text editors just display the same way? Atom seems messed up for some reason...using Sublime to fix

**Comment by timlevett** [comment]
:+1: :shipit:

**Pull request merged**